recipe/meta.yaml

```diff
@@ -13,33 +13,26 @@ source:
     - GEOSGeom_createCollection_vc9.patch  # [win and py27]
 
 build:
-  number: 1
-  skip: True  # [win and py36]
-  features:
-    - vc9  # [win and py27]
-    - vc10  # [win and py34]
-    - vc14  # [win and py>=35]
+  number: 2
+  run_exports:
+    # pretty poor backcompat.  SO name changes each release.
+    #   https://abi-laboratory.pro/tracker/timeline/geos/
+    - {{ pin_subpackage('geos', max_pin='x.x.x') }}
 
 requirements:
   build:
-    - python  # [win]
-    - msinttypes  # [win]
+    - msinttypes  # [win and vc<14]
     - cmake  # [win]
     - automake  # [not win]
     - libtool  # [not win]
-    - vc 9  # [win and py27]
-    - vc 10  # [win and py34]
-    - vc 14  # [win and py>=35]
-  run:
-    - vc 9  # [win and py27]
-    - vc 10  # [win and py34]
-    - vc 14  # [win and py>=35]
+    - {{ compiler('c') }}
```

Comment:
This needs to be avoided on Linux and OSX until all packages are built with the compiler.

Reply:
Indeed. I'll change that in a moment. Thanks @isuruf!

Reply:
I disagree.  I agree for core packages, but geos is pretty far outside of that, and it's a good place to understand any incompatibilities that may arise.  We had some luck early on with mixing old defaults and new.  It wasn't perfect, so it's not ideal, but we may get more mileage than you're expecting.